.github/workflows/ci-java.yml

```diff
@@ -123,6 +123,61 @@ jobs:
             bfd-server/bfd-server-war/target/server-work/access.*
             bfd-server/bfd-server-war/target/server-work/server-console.log
 
+  fhir-validate:
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Checkout repo'
+        uses: actions/checkout@v2
+      - name: 'Create local branch references'
+        # Check out the branches to build references for use in the next step
+        run: git fetch && git checkout master && git checkout $GITHUB_HEAD_REF
+      - name: 'Check for new/modified resources'
+        id: run_check
+        # Checks if there are any modified resources in the endpoint responses
+        # grep returns non-zero exit code when no matches are found, so only skip if the condition fails
+        run: |
+          can_skip=$(if [[ $(python3 ops/fhir_validations/diff_check.py) ]]; then echo "true"; else echo "false"; fi)
```

Comment:
If python3 is used in this step, do we even need the `Set up Python` step below this?